src/Components/Caesar/Caesar.js

```diff
@@ -40,32 +43,33 @@ class Caesar extends Component {
     }
     
     render() {
-        let plaintext = "Brutus homie what is up";
-        let ciphertext = "EUXWXV KRPLH ZKDW LV XS";
+        let plaintext = this.state.finalMessage;
+        let ciphertext = caesarShift(this.state.finalMessage, this.state.shift);
         let finalPlaintext, finalCiphertext;
         let fadein = [0,1]
         let bounce = [-50,0]
+        let newShift = -1*this.state.shift;
         if (this.state.animateMessage) {
             fadein = [1,1]
             bounce = [0,0]
-            finalPlaintext = <div className="overlay is-inline-flex">
+            finalCiphertext = <div className="overlay is-inline-flex">
                 <Anime 
                     rotateY={[0,45]} 
                     opacity={[1,0]} 
                     duration="200" 
                     delay={anime.stagger(50)}
                     easing="easeOutSine">
-                        {this.putCharsInSpans(plaintext)}
+                        {this.putCharsInSpans(ciphertext)}

```

Comment:
Another way of doing things, just if you'd like to remove a member function:

```suggestion
                        {ciphertext.map(c => <span>{c}</span>)}

```

Reply:
I'll keep Lisha's code here, because her putCharsInSpans function I believe creates the nice gradual effect of each letter being replaced-- straight mapping them replaces them almost all at once and doesn't look as ~special~ imo.